doc/source/user_guide/scale.rst

```diff
@@ -374,5 +374,34 @@ datasets.
 
 You see more dask examples at https://examples.dask.org.
 
+Use Modin
+---------
+
+Modin_ is a scalable dataframe library, which aims to be a drop-in replacement API for pandas and
+provides the ability to scale pandas workflows across nodes and CPUs available. It is also able
```

Comment:
Small thing, but maybe you can add the part about execution engines here, since in the next paragraph you say `Modin distributes computation across nodes and CPUs available` which seems to pretty much repeat the same thing again to conextualize that part?

Reply:
I wasn't able to rephrase the first and the second paragraphs by following your comment. To me the current state of paragraphs is pretty concise.

Reply:
To me, this intro paragraph aims to be a high level description of Modin. With that in mind, getting into execution engines here seems undesirable.